lib/salus/scanners/brakeman.rb

```diff
@@ -133,6 +133,9 @@ def user_supplied_ignore_expirations?
         return true if item.key?('expiration')
       end
       false
+    rescue TypeError, Errno::ENOENT => e
```

Comment:
Could you rescue `StandardError` instead, since it's a super class of `TypeError`?
That way, we could potentially catch more errors.

Reply:
Also, I'm not sure why the `rescue` includes `Error::ENOENT`, which means no file or directory.  
There is nothing in this function that could possibly throw this type of exception.